scripts/covid_tracking_project/historic_us_data/config.txt

```diff
@@ -0,0 +1,14 @@
+import: {
+    import_name: "COVIDTracking_US"
+    provenance_url: "https://www.covidtracking.com"
+    provenance_description: "COVID-19 related data for US from The COVID Tracking Project."
+    # Path to the resolved MCF.
+    mcf_url: "gs://resolved_mcf/external_tables/covid_tracking_us/nodes.mcf-00000-of-00001"
+    # First date of imported data.
+    start_date: "2020-01-22"
+    # Last date of imported data.
+    end_date: "2020-06-22"
+    # Source data release schedule.
+    update_frequency: "P1D"
+    next_release_date: "2020-06-23"
```

Comment:
Can remove end_date and next_release_date for now

Reply:
Fixed~